packages/alea-frontend/pages/sys-admin.tsx

```diff
@@ -1,43 +1,336 @@
-import { NextPage } from "next";
-import MainLayout from "../layouts/MainLayout";
-import { Button } from "@mui/material";
-import { recomputeMemberships } from "@stex-react/api";
-import { useRouter } from "next/router";
-import { useState } from "react";
+import { NextPage } from 'next';
+import MainLayout from '../layouts/MainLayout';
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  IconButton,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TextField,
+  Typography,
+} from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import CheckIcon from '@mui/icons-material/Check';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { getAuthHeaders, recomputeMemberships } from '@stex-react/api';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
+const SysAdmin: NextPage = () => {
+  const [aclId, setAclId] = useState<string | null>('');
+  const [resourceId, setResourceId] = useState<string | null>('');
+  const [actionId, setActionId] = useState<string | null>('');
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false);
+  const [error, setError] = useState('');
+  const [resourceActions, setResourceActions] = useState([]);
+  const [editing, setEditing] = useState<{
+    aclId: string;
+    actionId: string;
+    resourceId: string;
+  } | null>(null);
+  const [newAclId, setNewAclId] = useState<string | null>('');
+  const [deleteDialogOpen, setDeleteDialogOpen] = useState(false);
+  const [deleteResource, setDeleteResource] = useState<{
+    resourceId: string;
+    actionId: string;
+  } | null>(null);
 
-const SysAdmin : NextPage = () => {
-    const [recomputingMemberships, setRecomputingMemberships] = useState(false);
-    const router = useRouter();
-    async function handleRecomputeClick(){
-        try{
+  useEffect(() => {
+    async function getAllResources() {
+      try {
+        const { data } = await axios.get(
+          '/api/access-control/get-all-resourceacces-pairs',
+          { headers: getAuthHeaders() }
+        );
+        setResourceActions(data);
+      } catch (e) {
+        console.error(e);
+      }
+    }
+    getAllResources();
+  }, []);
 
-            setRecomputingMemberships(true);
```

Comment:
Don't remove this code. The button should be disabled while recomputing. To help the user understand that some action is going on. Otherwise the user will keep clicking unnecessarily.